src/providers/index.ts

```diff
@@ -0,0 +1,237 @@
+import chalk from 'chalk';
+import dedent from 'dedent';
+import fs from 'fs';
+import yaml from 'js-yaml';
+import path from 'path';
+import cliState from '../cliState';
+import logger from '../logger';
+import type { LoadApiProviderContext, TestSuiteConfig } from '../types';
+import type { EnvOverrides } from '../types/env';
+import type { ApiProvider, ProviderOptions, ProviderOptionsMap } from '../types/providers';
+import invariant from '../util/invariant';
+import { getNunjucksEngine } from '../util/templates';
+import { AI21ChatCompletionProvider } from './ai21';
+import { AnthropicCompletionProvider, AnthropicMessagesProvider } from './anthropic';
+import {
+  AzureAssistantProvider,
+  AzureChatCompletionProvider,
+  AzureCompletionProvider,
+  AzureEmbeddingProvider,
+} from './azure';
+import { BAMChatProvider, BAMEmbeddingProvider } from './bam';
+import { AwsBedrockCompletionProvider, AwsBedrockEmbeddingProvider } from './bedrock';
+import { BrowserProvider } from './browser';
+import { ClouderaAiChatCompletionProvider } from './cloudera';
+import { CohereChatCompletionProvider, CohereEmbeddingProvider } from './cohere';
+import { FalImageGenerationProvider } from './fal';
+import { GolangProvider } from './golangCompletion';
+import { GoogleChatProvider } from './google';
+import { GroqProvider } from './groq';
+import { HttpProvider } from './http';
+import {
+  HuggingfaceFeatureExtractionProvider,
+  HuggingfaceSentenceSimilarityProvider,
+  HuggingfaceTextClassificationProvider,
+  HuggingfaceTextGenerationProvider,
+  HuggingfaceTokenExtractionProvider,
+} from './huggingface';
+import { LlamaProvider } from './llama';
+import {
+  LocalAiCompletionProvider,
+  LocalAiChatProvider,
+  LocalAiEmbeddingProvider,
+} from './localai';
+import { ManualInputProvider } from './manualInput';
+import { MistralChatCompletionProvider, MistralEmbeddingProvider } from './mistral';
```

Comment:
Nice change. Instead of importing everything here _and_ in the registry, is there a one or two-line approach where the registry does the exports and we just pass them through?